.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: php
+
+php:
+  - '5.6'
+  - '7.0'
```

Comment:
Please also test PHP 7.1

Reply:
I did, but had to remove it because of `PHPUnit\Framework\Exception: PHP Fatal error:  strict_types declaration must be the very first statement in the script in /home/travis/.phpenv/versions/7.1.10/bin/phpunit on line 2` - https://travis-ci.org/hellofresh/stats-php/jobs/289517498

Reply:
I think you can solve this by using your own version of PHPUnit not the one from travis since it is out of date (aka PHPUnit 6.4.1 [fixes](https://github.com/sebastianbergmann/phpunit/issues/2791) this issue)

Reply:
`phpunit/phpunit 6.4.3 requires php ^7.0` and we need to support 5.6

Reply:
I know and you are using 6.4.0 since that is what is installed on travis (see [line 11](https://github.com/hellofresh/stats-php/pull/1/files#diff-354f30a63fb0907d4ad57269548329e3R11)) you probably want to call your installed version `vendor/bin/phpunit` instead. 

To my knowledge this bug was introduced in phpunit 6.4.0 so 4.8.0 should work fine

Reply:
Works fine with `5.7.23` and `5.7.0`, so added 7.1 back.